ruma-client-api/src/r0/keys/get_keys.rs

```diff
@@ -45,10 +45,27 @@ ruma_api! {
         /// If any remote homeservers could not be reached, they are recorded
         /// here. The names of the properties are the names of the unreachable
         /// servers.
+        #[serde(skip_serializing_if = "BTreeMap::is_empty")]
```

Comment:
Timo had these added, just wanted to make sure you saw.

Reply:
If we do `skip_serializing_if`, there should also be `serde(default)`. Otherwise seems correct.